Assets/Scripts/EchoLocation/ReplacementShaderCam.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+[ExecuteInEditMode]
+public class ReplacementShaderCam : MonoBehaviour {
+    public Shader ReplacementShader;
+    void OnEnable()
+    {
+        if (ReplacementShader != null)
+        {
+            GetComponent<Camera>().SetReplacementShader(ReplacementShader, "");
+        }
+    }
+
+    // Update is called once per frame
```

Comment:
Is OnDisable an Update? Is it called once per frame?

Reply:
nah, forgot to delete the comment.